comp/dogstatsd/server/enrich.go

```diff
@@ -84,6 +85,10 @@ func extractTagsMetadata(tags []string, originFromUDS string, originFromMsg []by
 			continue
 		} else if strings.HasPrefix(tag, jmxTagPrefix) {
```

Comment:
On further thought, we should remove the old way of checking jmx in favor of this new tag.
so lets get rid of `jmxTagPrefix` and the associated check altogether. It doesn't offer us anything that `jmx_chec_name` doesn't.